### PR TITLE
zip: 64-bit only

### PR DESCRIPTION
**Pull request by Mno-hime**
The same trick as for the `unzip` PR.

### CODE REVIEW

File: components/archiver/zip/Makefile [hunk 1]
```diff
@@ -36,32 +37,32 @@ COMPONENT_BUGDB=	utility/zip
...
+build:		$(BUILD_64)
 
-install:	$(INSTALL_32)
+install:	$(INSTALL_64)
 
 test:		$(NO_TESTS)
 
 BUILD_PKG_DEPENDENCIES =	$(BUILD_TOOLS)
 
-include $(WS_TOP)/make-rules/depend.mk
```
Comment (alarcher) [inline components/archiver/zip/Makefile]: Can you remove this line?

File: components/archiver/zip/Makefile [hunk 2]
```diff
@@ -36,32 +37,32 @@ COMPONENT_BUGDB=	utility/zip
...
 
 # remove the unwanted file.
 COMPONENT_PREP_ACTION = rm -f $(SOURCE_DIR)/match.S
 
 # The Makefile we want is under unix/Makefile
-COMPONENT_BUILD_ARGS += -f unix/Makefile
-COMPONENT_BUILD_TARGETS += generic CC=$(CC)
-COMPONENT_INSTALL_ARGS += -f unix/Makefile
+COMPONENT_BUILD_ARGS    += -f unix/Makefile
+COMPONENT_BUILD_TARGETS += generic CC="$(CC) -m64"
```
Comment (alarcher) [inline components/archiver/zip/Makefile]: Could use the CC_BITS macros

Comment (Mno-hime) [reply components/archiver/zip/Makefile]: Maybe this works only for autotools? With `CC_BITS = -m64` it build as 32-bit:
```
/usr/gcc/6/bin/gcc -c -I. -DUNIX -O3 -DBZIP2_SUPPORT -DUIDGID_NOT_16BIT -DLARGE_FILE_SUPPORT -DUNICODE_SUPPORT -DHAVE_DIRENT_H -DHAVE_TERMIOS_H zip.c
```

Comment (alarcher) [reply components/archiver/zip/Makefile]: I meant to replace -m64 with $(CC_BITS)

**Pull request merged**